### PR TITLE
Add licecap

### PR DESCRIPTION
**Pull request by nusr**
Add licecap.
LICEcap is a simple animated screen capture tool.
## Project URL

https://github.com/justinfrankel/licecap
## Category

Images
## Description

 LICEcap simple animated screen capture tool for Windows and OS X .
## Why it should be included to `Awesome macOS open source applications ` (optional)
It's small.

## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [ ] Has a **clear** README in English

### CODE REVIEW

File: applications.json [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+        "Images"
+      ],
+      "repo_url" : "https://github.com/justinfrankel/licecap",
+      "title" : "licecap",
+      "icon_url" : "",
+      "screenshots" : [
+        "https://www.cockos.com/licecap/demo2.gif",
+        "https://www.cockos.com/licecap/how_to_licecap.gif"
+      ],
+      "official_site" : "",
```
Comment (serhii-londar) [inline applications.json]: Add "official_site" : "http://www.cockos.com/licecap" .

Comment (nusr) [reply applications.json]: I make it.

File: applications.json [hunk 2]
```diff
@@ -1,5 +1,22 @@
 {
   "applications" : [
+    {
+      "short_description" : "LICEcap simple animated screen capture tool for Windows and OS X .",
+      "categories" : [
+        "Images"
```
Comment (serhii-londar) [inline applications.json]: I'm not sure it will work with a capitalized category name.

Comment (nusr) [reply applications.json]: @serhii-londar  It only produce gif file.

Comment (serhii-londar) [reply applications.json]: I'm talking about:
"categories" : [
        "Images"
]

Comment (serhii-londar) [reply applications.json]: Can you change to 
"categories" : [
        "images"
]
?

Comment (nusr) [reply applications.json]: @serhii-londar I got it.

**Comment by serhii-londar** [comment]
@nusr This repo does not have Readme.md file.

**Pull request closed without merging**